src/qt/forms/sendcoinsdialog.ui

```diff
@@ -848,7 +848,7 @@
                  <item>
                   <widget class="QLabel" name="labelCustomPerKilobyte">
                    <property name="toolTip">
-                    <string>If the custom fee is set to 1000 satoshis and the transaction is only 250 bytes, then &quot;per kilobyte&quot; only pays 250 satoshis in fee, while &quot;total at least&quot; pays 1000 satoshis. For transactions bigger than a kilobyte both pay by kilobyte.</string>
+                    <string>Specify a custom fee per 1,000 bytes of the virtual transaction's size.</string>
```

Comment:
The transaction is not virtual. The size is...?

```
transation's virtual size.

Reply:
Without the typo:

    Specify a custom fee per 1,000 bytes of the transaction's virtual size.